src/Manifest/Manifest.router.ts

```diff
@@ -36,6 +36,15 @@ export class ManifestRouter extends Router {
       server.handleRequest(this.getProjectManifest)
     )
 
+    /**
+     * Returns the manifest of a land associated project  (Builder In World)
+     */
+    this.router.get(
+      '/projects/:land/manifestFromCoordinates',
```

Comment:
Careful here! remember that using REST means doing_something like_:
`resource/ID/action`
but :land here is not indexing project.

Maybe we can add a query string filter to the already existing `projects/:id/manifest` route? @cazala

Reply:
Sure let me know how do you want it and I will change it! An example to help will be cool!

Reply:
The thing with `/projects/:id/mainfest` is that it requires you to know the project ID, which is not the case here. In order to keep this RESTful, maybe we can just use the endpoint `/manifests` with a `created_location` query param, like `/manifests?created_location=10,10`.

Reply:
oh yes, thought of that afterwards but wasn't sure if the id was available. I like the idea of having a `/manifests` endpoint with a query param.
The only problem I see with it is that you **have** to have access to the project(s) returned by it. So the idea would be to return all manifests you have access to, filtering by the query string.

Regarding [this comment](https://github.com/decentraland/builder-server/pull/197#discussion_r611717182) I see that it's [still a thing](https://github.com/decentraland/builder-server/pull/197/files#diff-536f13249592afdba56fe4ccd5ba79296b564dcc392bbd5049f46509ece9e6e0R95) (with the method name). What do you guys think of using `coordinates` or `land_coordinates` everywhere?
I think coordinates is already a concept we use in parcel-related projects

@Suduck @cazala

Reply:
It is fine by me! let me know which one you prefer and I will change it in all parts!

Reply:
Ok after talking a bit with juan, it's a hard name to do, but what do you think of `creation_coords` ? and to use that for the query string @Suduck 

if this is time-pressing we can go with the current implementation and change it later

Reply:
Sounds good to me! I will change it now since it will be easier because if we change it later we will need to sync kernel and unity master branches at the same time to ensure it is working. 

I will make the changes and ask for a revision again!

Reply:
Thanks a lot! sorry for the back and forth!